tests/multipleBackend/routes/routeBackbeat.js

```diff
@@ -306,14 +312,15 @@ describe('backbeat routes', () => {
             assert.strictEqual(StorageClass, 'STANDARD');
         }
 
-        beforeEach(() =>
-            bucketUtil.emptyIfExists(BUCKET_FOR_NULL_VERSION)
-                .then(() => s3.createBucket({ Bucket: BUCKET_FOR_NULL_VERSION }).promise())
-        );
+        beforeEach(() => {
+            bucket = `${BUCKET_FOR_NULL_VERSION_PREFIX}-${Date.now()}`;
```

Comment:
Date.now() usually causes flaky tests, unless you are confident I would suggest to use UUID

Reply:
right, fixed thanks :+1: